mtgbp/assetlist.json

```diff
@@ -0,0 +1,45 @@
+{
+    "$schema": "../assetlist.schema.json",
+    "chain_name": "mtgbp",
+    "assets": [
+      {
+        "description": "The native staking and governance token of mtgbp.",
+        "extended_description": "mtgbp is a Layer 1 protocol that allows developers to automatically spin up VM-agnostic, parallelized and interoperable dedicated chains, or “Chainlets,” that provide applications with infinite horizontal scalability. Each Chainlet is a replica of the mtgbp Mainnet, with the same validator set and security model.\n\nmtgbp’s mission is to enable the next 1000 chains in gaming and entertainment as part of the growing mtgbp Multiverse.",
```

Comment:
These all have the same extended_description. What is the relationship between these three chains? Why are there 3 of the same thing but with different names?

Reply:
These are three chains for three different clients who wants their own blockchains. We are just learning the cosmos blockchain and cosmos SDK so we are unfortunately rookies in the system. 

We have set the parameters for our clients, it’s standardized from our side. We are learning as we go.

This is why the three chains have the same descriptions, this has not been modified by us (it’s the cosmos sdk standard description) and we missed to change this.

Thanks for pointing it out, we will change it and we will see if we find other places that needs personalizations in the code. @JeremyParish69